.cirrus.yml

```diff
@@ -53,11 +53,15 @@ task:
       pkg install -y
       bison
       cmake
+      dbus
       flex
+      gcc12
       git
       libgit2
       llvm12
       ninja
+      pkgconf
+      xfce4-conf
```

Comment:
This is probably not needed if xfconf plugin gets disabled anyway?

Reply:
This should be the case, but I am not sure to remove it because if it is not installed, the xfconf plugin would not be part of the CI in FreeBSD anymore.